dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -58,8 +58,6 @@ export type Props = {
 	showMainVideo?: boolean;
 	kickerText?: string;
 	showPulsingDot?: boolean;
-	/** Sometimes kickers and headlines are separated by a slash */
-	showSlash?: boolean;
 	/** Sometimes kickers and headlines are separated by a linebreak */
 	hideLineBreak?: boolean;
```

Comment:
I wonder whether we should remove this as a prop from the Card component? As things stand, most uses of `Card` are wrapped in `FrontCard`, which doesn't have the prop on it. And I'm not aware of any cases where a card shouldn't have a linebreak between kicker and title for its main headline (as opposed to for its sublinks). What do you think?

Reply:
Brilliant! Yep, I'll remove it, the fewer props the better.